pkg/controller/job/jobcontroller.go

```diff
@@ -167,7 +173,7 @@ func (jm *JobController) getPodJob(pod *api.Pod) *batch.Job {
 		return nil
 	}
 	if len(jobs) > 1 {
-		glog.Errorf("user error! more than one job is selecting pods with labels: %+v", pod.Labels)
+		utilruntime.HandleError(fmt.Errorf("user error! more than one job is selecting pods with labels: %+v", pod.Labels))
```

Comment:
This is for the manual label selector which will soon be gone when I remove extensions.Job. Backwards compatibility, mainly.